magento/pwa-studio
# Ignoring CE and EE gql files based on Mangeto edition.

## Description

Ignore CE and EE files based on `env.MAGENTO_BACKEND_EDITION`.

## Related Issue
Closes PWA-843

## Acceptance
`yarn run build:dev` should work fine.

### Verification Stakeholders
@dpatil-magento 

### Verification Steps
1. Checkout this branch.
2. Use a valid CE Magento backend and set MAGENTO_BACKEND_EDITION to CE. Run `yarn build:dev`, it should work fine.
3. Use a valid EE Magento backend and set MAGENTO_BACKEND_EDITION to EE. Run `yarn build:dev`, it should work fine.

## Screenshots / Screen Captures (if appropriate)

![image](https://user-images.githubusercontent.com/35203638/110169203-fe14d080-7dbd-11eb-9d35-16c24bb7d6c8.png)

## Checklist
* I have added tests to cover my changes, if necessary.
* I have added translations for new strings, if necessary.
* I have updated the documentation accordingly, if necessary.
